Makefile

```diff
@@ -0,0 +1,28 @@
+########################################################################################
+
+# This Makefile generated by GoMakeGen 0.5.0 using next command:
+# gomakegen --output Makefile .
+
+########################################################################################
+
+.PHONY = fmt all clean deps
+
+########################################################################################
+
+all: vgrepo
+
+vgrepo:
+	go build vgrepo.go
+
+deps:
+	git config --global http.https://pkg.re.followRedirects true
+	go get -d -v pkg.re/essentialkaos/ek
```

Comment:
This is looks strange. This record means that you use somewhere ek import without version.